src/data/list/basic.lean

```diff
@@ -2327,6 +2327,10 @@ theorem map_pmap {p : α → Prop} (g : β → γ) (f : Π a, p a → β)
   (l H) : map g (pmap f l H) = pmap (λ a h, g (f a h)) l H :=
 by induction l; [refl, simp only [*, pmap, map]]; split; refl
 
+theorem pmap_map {α β γ} {p : β → Prop} (g : ∀ b, p b → γ) (f : α → β)
+  (l H) : pmap g (map f l) H = pmap (λ a h, g (f a) h) l (λ a h, H _ (mem_map_of_mem _ h)) :=
```

Comment:
Could you add the types for `l` and `H`, for readability? (I know I'm being a pot-calling-a-kettle-black, but it is a good idea!)

Reply:
Also, I think `α β γ` are already defined as variables, so you don't need them here.